src/rest/Query.ts

```diff
@@ -15,6 +15,16 @@ export interface IFacetOptions {
   freezeFacetOrder?: Boolean;
 }
 
+/**
+ * The information about the user we'd like to check and its actions.
+ */
+export interface IUserActions {
```

Comment:
Maybe a better name here might be `IUserActionsRequest`? I think we should use a singular name since it is not an array.

Reply:
👍